roles/test_observabilityStrategy/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+# Assuming we've already logged in...
+# Following procedure on https://infrawatch.github.io/documentation/#configuring-observability-strategy_assembly-installing-the-core-components-of-stf
+# Assuming we're in teh right project already...
+
+
+- name: "Add observabilityStrategy: none to the spec section"
+  ansible.builtin.shell:
+    cmd: |
+      oc patch stf/default --type merge -p '{"spec":{"observabilityStrategy": "none"}}' && sleep 120
+  changed_when: false
+
+
+- name: "Delete Grafana object"
+  ansible.builtin.shell:
+    cmd: |
+      oc delete grafana default && sleep 60
```

Comment:
These sleeps make me a bit nervous because they are unevenly applied. For example, the time between "Delete Grafana object" and "Verify that Grafana object didn't respawn" is >= 4 mins, whereas the time between  "Delete ElasticSearch object" and "Verify that elasticsearch object didn't respawn" is `4 * <get pods time>` which hopefully is a lot less than 4 mins.

I think ultimately what you're trying to do here is wait long enough to ensure that the pods actually disappeared, and then additional time to make sure they don't come back. 

One approach that comes to mind is to replace the `sleep 60`s with something like `while oc get po -l app=grafana | grep grafana; do sleep 1; done;`. This would actually ensure the pod was gone, and would usually take less time than the 60s sleep, too. Then you'd be sure that each pod disappeared, and could do a single sleep (even up to that original 4 minutes if you think it's necesarry) before checking to ensure they don't return.

Reply:
Thanks for the comment. I made the changes.